includes/class-wcs-template-loader.php

```diff
@@ -289,4 +357,53 @@ public static function get_opc_subscription_add_to_cart() {
 		global $product;
 		wc_get_template( 'checkout/add-to-cart/simple.php', array( 'product' => $product ), '', PP_One_Page_Checkout::$template_path );
 	}
+
+	/**
+	 * Handles relocated templates.
+	 *
+	 * Hooked onto 'wc_get_templates'.
+	 *
+	 * @since 1.4.0
```

Comment:
Is the number for @since the subscriptions-core's version or WCS? I assume subscriptions-core's but looking at the other functions above, seems like it refers to WCS' version.

Reply:
Yeah, we should probably use the subscriptions-core version in `@since` for doc-block. Those other functions are copy-pasted from WCS and never updated. 

No huge priority to get those updated at this stage